docs/Commands.md

```diff
@@ -437,6 +438,14 @@ FT.SEARCH books-idx "python" RETURN 3 $.book.price AS price
 
 - **SORTBY {attribute} [ASC|DESC]**: If specified, the results
   are ordered by the value of this attribute. This applies to both text and numeric attributes.
+
+- **MULTISORTBY {nargs} {property} {ASC|DESC} [MAX {num}]**: Sort the pipeline up until the point of MULTISORTBY,
```

Comment:
I suggest to use the same terminology.
We use `{property}`, `sorting parameters` (in the documentation details), and `{attribute}` (in `SORTBY`)

Reply:
Good catch. This is copy-pasted from SORTBY in FT.AGGREGATE.
Should we change it there as well? @emmanuelkeller